docs/_docs/04_community/contributing.md

```diff
@@ -166,10 +167,14 @@ We have a separate CI pipeline for Azure code. To run it on a forked repo:
 
 1. Create a new `Secret` named `AZURE_CREDENTIALS` and paste the Azure credentials you copied from the 1<sup>st</sup> step as the value
 
+    > `AZURE_CREDENTIALS` will be stored on _your GitHub account_. GitHub stores your secrets, encrypted an in a very secure place ([GitHub Actions Secrets Reference](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets)). Once the secret created, it's only possible to update or delete it, value of the secrets can't be viewed. GitHub uses a [libsodium sealed box](https://libsodium.gitbook.io/doc/public-key_cryptography/sealed_boxes) to help ensure that secrets are encrypted before they reach GitHub.
+
 1. Create a [new Personal Access Token (PAT)](https://github.com/settings/tokens/new) page under [Settings](https://github.com/settings/profile) / [Developer Settings](https://github.com/settings/apps), making sure `write:discussion` and `public_repo` scopes are checked. Click the _Generate token_ button and copy the generated PAT.
 
 1. Go back to settings/secrets in your fork and [Create a new Secret](https://docs.github.com/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) named `PAT`.  Paste the output from the 4<sup>th</sup> step as the value
 
+    > `PAT` will be stored on _your GitHub account_. GitHub stores your secrets, encrypted an in a very secure place ([GitHub Actions Secrets Reference](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets)). Once the secret created, it's only possible to update or delete it, value of the secrets can't be viewed. GitHub uses a [libsodium sealed box](https://libsodium.gitbook.io/doc/public-key_cryptography/sealed_boxes) to help ensure that secrets are encrypted before they reach GitHub.

```

Comment:
```suggestion
    > `PAT`will be stored on _your GitHub account_. GitHub stores your secrets, encrypted in a secure location.  See ([GitHub Actions Secrets Reference](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets)) for more detailed information.

```

Reply:
contributing guideline updated, thanks for the feedback